docs/Catalog/RemovingStacks.md

```diff
@@ -0,0 +1,61 @@
+<!-- markdownlint-disable-next-line first-line-h1 -->
+From time to time, you may need to remove a Quilt stack from your AWS account.
+To do this, you will need to delete the CloudFormation template that created the
+stack, which will remove all resources associated with the stack.
+
+### NOTE: Pausing the Stack Instead of Deletion
+
+If you prefer to suspend the Quilt stack temporarily, consider pausing services
+(e.g., shutting down ECS) to reduce costs. Note, however, that some charges may
+continue even with ECS inactive. To fully stop charges, you must delete the
+CloudFormation stack.
+
+## Step 1: Confirm Stack Deletion Readiness
+
+Deleting the CloudFormation stack will permanently remove any stack-specific
+configurations, analytics data, audit logs, and Athena querying setups. Data in
+your own S3 buckets will remain intact, as they are outside of CloudFormation’s
+control.
+
```

Comment:
Should we describe taking a back up of the database to capture user state. We can only restore from a database backup using a terraform deployment.

If we don't go into that level of detail, we should probably at least mention that user permission mappings will be lost when users tear down their stack.

Reply:
Added